editor/tests/org.fusesource.ide.wsdl2rest.tests/src/org/jboss/fuse/wsdl2rest/test/doclit/CamelCxfWsDocLitTest.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.fuse.wsdl2rest.test.doclit;
+/*
+ * Copyright (c) 2008 SL_OpenSource Consortium
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import static org.jboss.fuse.wsdl2rest.test.Item.DATE_FORMAT;
+
+import java.net.URL;
+import java.util.Date;
+
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.ServiceStatus;
+import org.jboss.fuse.wsdl2rest.test.Item;
+import org.jboss.fuse.wsdl2rest.test.ItemBuilder;
+//import org.jboss.fuse.wsdl2rest.util.SpringCamelContextFactory;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CamelCxfWsDocLitTest {
+
+//    @Test
```

Comment:
completely commented out test class, Should we remove it or uncomment it?

Reply:
Removed whole test plugin